TemsApi/src/Domain/Entities/Asset.cs

```diff
@@ -2,7 +2,7 @@
 {
     public class Asset
     {
-        public Guid Id { get; set; }
+        public Guid Id { get; set; } = Guid.NewGuid();
```

Comment:
remove this  = Guid.NewGuid();
It should not be present here.
The ID is generated by the repository upon creation